moped-editor/src/components/GridTable/GridTable.js

```diff
@@ -222,17 +222,23 @@ const GridTable = ({ title, query }) => {
       return;
     }
 
-    // If the operator includes "is_null", then the value is always true
+    // If the operator includes "is_null", we check for empty strings
     if (gqlOperator.includes("is_null")) {
-      value = "true";
+      gqlOperator = envelope ? "_eq" : "_neq";
```

Comment:
Making envelope a boolean attribute seems to be an anti-pattern. I think it should remain as string type to wrap up values going into the database. I'd be ok if we switch the boolean value to something like `envelope === "true"`.

Reply:
works for me -- I originally had it as envelope === true but it felt weird. 
do we ever use the string true to wrap a value?

Reply:
ill change it back to the envelope being a string